### PR TITLE
Add Vagrant setup.

### PR DESCRIPTION
**Pull request by shishir-a412ed**

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
ping @rhatdan for review.

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Specify minimum Vagrant version and Vagrant API version
+Vagrant.require_version ">= 1.6.0"
+VAGRANTFILE_API_VERSION = "2"
+
+# Create box
+Vagrant.configure("2") do |config|
+  config.vm.define "docker-lvm-plugin-fedora32"
```
Comment (rhatdan) [inline Vagrantfile:7]: Can you update this to fedora 33. Fedora 32 will be out of date in June.

Comment (shishir-a412ed) [reply Vagrantfile]: Updated to fedora 33.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**